src/app/pages/Designathons/Designathon24/Designathon24.module.scss

```diff
@@ -1,10 +1,16 @@
+@import url("https://fonts.googleapis.com/css2?family=DM+Sans:wght@400;500;700&display=swap");
+
 .container {
 	--des24-pink: #ff75a4;
 	--des24-hot-pink: #f84e72;
 	--des24-peach: #ff6969;
 	--des24-cream: #fff5e0;
 	--des24-black: #242424;
 
+	* {
+		font-family: "DM Sans";
+	}
```

Comment:
using wildcard like this makes your css extremely unperformant and needs be carefully used

Reply:
I'm not sure how to apply DM Sans to all of the Designathon24 pages in one place other than with the wildcard selector. Can you point me towards how you'd prefer it to be done?

Reply:
Our ducimain font should already be "DM Sans", so I don't think you need to do anything particular for the Designathon pages. However, agreed that we should reduce the overall use of universal selectors, notably since the current main site does such in not the best way.

Reply:
_Just realized that `ducimain` stands for "Design UCI Main"_ 🤣 

I believe the current ducimain font is of a bolder varietal, and font-weight doesn't affect it, so default "DM Sans" actually does look noticeably lighter.

<img width="1361" alt="Screenshot 2024-03-26 at 11 47 38 AM" src="https://github.com/designatuci/DUCI-website/assets/100006999/bc509fe7-5aca-4408-984e-103e4a6cdc85">

Ducimain font ^^


![chrome-capture-2024-2-26](https://github.com/designatuci/DUCI-website/assets/100006999/90fd5634-f2b2-4078-81ad-fa46f9a77b8d)

Swapping between Ducimain and DM Sans ^^

Reply:
Ahh okay, it seems the existing file includes only medium and bold weights with a different stylistic set (notably characters `a`, `g`, `Q`) and with a wider kerning. Could we omit any typeface changes for now and revisit this after discussion with the designers?

Reply:
Done 🫡